doc/source/groupby.rst

```diff
@@ -268,6 +269,8 @@ natural to group by one of the levels of the hierarchy.
    index = pd.MultiIndex.from_tuples(tuples, names=['first', 'second'])
```

Comment:
This is actually a bit a convoluted way to make the multiIndex. The arrays could also be passed directly to `MultiIndex.from_arrays`.